demos/python_demos/rag_chatbot/servable_stream/ov_embedding_model.py

```diff
@@ -45,20 +47,28 @@ class OVEmbeddings(BaseModel, Embeddings):
     @classmethod
     def from_model_id(
         cls,
-        model_id: str,
+        model_name: str,
         do_norm: bool,
         ov_config: Optional[dict],
         model_kwargs: Optional[dict],
         **kwargs: Any,
     ):
         _model_kwargs = model_kwargs or {}
         _ov_config = ov_config or {}
-        tokenizer = AutoTokenizer.from_pretrained(model_id, **_model_kwargs)
-        core = ov.Core()
-        model_path = Path(model_id) / "openvino_model.xml"
-        model = core.compile_model(model_path, **_ov_config)
-        num_stream = model.get_property('NUM_STREAMS')
+        if os.path.isdir(model_name):
```

Comment:
Is it the same as in original notebook? Or where is the change from?

Reply:
this is the path passed in env variable LLM_MODEL_DIR or passed as model name from HF